packages/KSkeleton/KSkeleton.vue

```diff
@@ -104,3 +103,8 @@ export default {
   }
 }
 </script>
+<style>
+.transparent .box {
+  background: transparent;
```

Comment:
can you rename class to `bg-transparent`, add it to _utils.scss and remove this style block?

Reply:
bonus points for animating their entry into the dom 😎

Reply:
I'm not sure I can remove this style blocks as its needs the grandchild selector `.box` in order to alter the skeleton sub-sub-component skeletonbox styling. I may be mistaken.